spec/dummy/db/schema.rb

```diff
@@ -111,7 +111,8 @@
     t.boolean  "required",                  default: false
   end
 
-  add_index "fae_files", ["fileable_id", "fileable_type"], name: "index_fae_files_on_fileable_id_and_fileable_type", using: :btree
```

Comment:
I had to clean out and run a migration on the database from scratch. I believe that the changes  are just the order of where they appear in the schema.